src/main/resources/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorWorkflowStep/config.groovy

```diff
@@ -46,8 +47,28 @@ f.section(title: descriptor.displayName) {
     f.select()
   }
 
-  f.entry(title: _(Messages.IqPolicyEvaluation_Application()), field: 'iqApplication') {
-    f.select()
+  //Instance is null if this is a new Item.  We need to set a default.
+  String manualAppId = ''
```

Comment:
Why does this deviate from `IqPolicyEvaluatorBuildStep/config.groovy`?

Reply:
>Why does this deviate from `IqPolicyEvaluatorBuildStep/config.groovy`
 
Changed 50078e3